spec/features/finance/npq/course_payment_breakdown_spec.rb

```diff
@@ -171,8 +169,8 @@ def then_i_should_see_correct_output_payment_breakdown
     within first(".app-application__card") do
       expect(page).to have_css("tr:nth-child(1) td:nth-child(1)", text: "Output payment")
       expect(page).to have_css("tr:nth-child(1) td:nth-child(2)", text: total_declarations(npq_leading_behaviour_culture_contract))
-      expect(page).to have_css("tr:nth-child(1) td:nth-child(3)", text: number_to_pounds(160))
-      expect(page).to have_css("tr:nth-child(1) td:nth-child(4)", text: number_to_pounds(1440))
+      expect(page).to have_css("tr:nth-child(1) td:nth-child(3)", text: number_to_pounds(162))
```

Comment:
i've checked these by hand. these were different as we were lazy loading the contracts and were never evaluated therefore `first(".app-application__card")` would have been for a different contract/course